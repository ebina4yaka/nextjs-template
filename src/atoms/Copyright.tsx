import { ReactElement } from 'react'

export default function Copyright(): ReactElement {
  return (
    <div className="font-sans text-base">
      {'Copyright Â© '}
      <a
        href="https://github.com/ebina4yaka"
        target="_blank"
        rel="noreferrer noopener"
      >
        ebina4yaka
      </a>
      {` ${new Date(Date.now()).getFullYear()}.`}
    </div>
  )
}
