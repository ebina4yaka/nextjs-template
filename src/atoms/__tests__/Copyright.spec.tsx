import renderer from 'react-test-renderer'
import Copyright from '../Copyright'

Date.now = jest.fn(() => 1482363367071)

it('Snapshot test for Copyright', () => {
  const tree = renderer.create(<Copyright />).toJSON()
  expect(tree).toMatchSnapshot()
})
