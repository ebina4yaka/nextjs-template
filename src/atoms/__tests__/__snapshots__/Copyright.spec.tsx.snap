// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Copyright 1`] = `
<div
  className="font-sans text-base"
>
  Copyright Â© 
  <a
    href="https://github.com/ebina4yaka"
    rel="noreferrer noopener"
    target="_blank"
  >
    ebina4yaka
  </a>
   2016.
</div>
`;
